3.10/release-notes-api-changes310.md

```diff
@@ -15,6 +15,12 @@ integrations for ArangoDB 3.10.
 
 ### Endpoint return value changes
 
+Since ArangoDB 3.8, there were two APIs for retrieving the metrics in two different formats: `/_admin/metrics` and `/_admin/metrics/v2`. The metrics API v1 (`/_admin/metrics`) was deprecated in 3.8 and usage of `/_admin/metrics/v2` was encouraged.  
```

Comment:
```suggestion
Since ArangoDB 3.8, there have been two APIs for retrieving the metrics in two different formats: `/_admin/metrics` and `/_admin/metrics/v2`. The metrics API v1 (`/_admin/metrics`) was deprecated in 3.8 and usage of `/_admin/metrics/v2` was encouraged.  
```